examples/nest/hpc_benchmark.sli

```diff
@@ -41,23 +41,64 @@
 
 /nvp 1 def % total number of virtual processes
 /scale 0.2 def %  scaling factor of the network size, total network size = scale*11250 neurons
-/Tsim 100. def % total simulation time in ms
+/simtime 1000. def % total simulation time in ms
+/presimtime 100. ms def % simulation time until reaching equilibrium
+/dt         0.1 ms def % simulation step
 /record_spikes true def % switch to record spikes of excitatory neurons to file
 /path_name (.) def % path where all files will have to be written
 /log_file (log) def % naming scheme for the log files
 
 % -------------------------------------------------------------------------------------
 
+%%% Define function to convert synapse weight from  mV to pA %%%%%%%%%%
+
+/ConvertSynapseWeight
+{
+  /tauMem Set
+  /tauSyn Set
+  /CMem Set
+  % This function is specific to the used neuron model
+  % Leaky integrate-and-fire neuron with alpha-shaped
+  % postsynaptic currents
+  (
+  a = tauMem / tauSyn;
+  b = 1.0 / tauSyn - 1.0 / tauMem;
+  % time of maximum
+  t_max = 1.0/b * (-LambertWm1(-exp(-1.0/a)/a)-1.0/a);
+  % maximum of PSP for current of unit amplitude
+  exp(1.0)/(tauSyn*CMem*b) * ((exp(-t_max/tauMem) - exp(-t_max/tauSyn)) / b - t_max*exp(-t_max/tauSyn))
+  ) ExecMath
+  1 exch div
+}
+def
+
+%%% Rise time of synaptic currents
+
+% The synaptic rise time is chosen such that the rise time of the
+% evoked post-synaptic potential is 1.700759 ms.
+% For alpha-shaped postynaptic currents, the rise time of the post-synaptic
+% potential follows from the synaptic rise time as
+
+/PSP_rise_time
+{
+  /taumem set
+  /tausyn set
+  
+  (
+  a = taumem / tausyn;
+  b = 1.0 / tausyn - 1.0 / taumem;
+  ) execmath
+  
+} def
+
+% Inverting this equation leads to tau_syn = 0.32582722403722841 ms, as specified in model_params below.
```

Comment:
This comment does not make sense to me (anymore), as you remove the magic number in line 88 to 90 below. Can you please change it to point to the use of the `ConvertSynapseWeight` and `PSP_rise_time` functions or rather remove it altogether? I am happy with removing it.